### PR TITLE
Add required deps and docs about lago for ubuntu

### PR DESCRIPTION
**Pull request by gbenhaim**
1. Added a list of deps from PyPi.
2. Added a list of .deb deps.
3. Added some documentation and a TODO list (see TODO).

Signed-off-by: gbenhaim galbh2@gmail.com

### CODE REVIEW

**Comment by gbenhaim** [comment]
First step in the way of making lago available on ubuntu.

File: install_scripts/ubuntu/TODO [hunk 1]
```diff
@@ -0,0 +1,36 @@
+README
+--------
+
+'install_deb_deps.sh' contains all the deps the aren't on PyPi
```
Comment (eedri) [inline install_scripts/ubuntu/TODO]: s/the/that

File: install_scripts/ubuntu/TODO [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+1. Find a solution for the creation of:
+    /var/lib/lago
+    /etc/lago.d
+
+
+Others
+--------
+
+1. Write an install script for ubuntu - There are to many changes that need to be made
```
Comment (eedri) [inline install_scripts/ubuntu/TODO]: s/to/too

**Comment by eedri** [comment]
Great start! 
Now you need an Ubuntu slave on CI so you can also run tests :)

**Comment by ovirt-infra** [comment]
Build finished.

**Comment by nvgoldin** [comment]
Closing for now, for some reason it keeps triggering CI.

**Pull request closed without merging**